staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -470,6 +471,8 @@ type DeleteOptions struct {
 
 	// Must be fulfilled before a deletion is carried out. If not possible, a 409 Conflict status will be
 	// returned.
+	// Converting url.Values to Preconditions requires manual conversion.
```

Comment:
hmm... this made this comment show up in API docs, which isn't really helpful... can drop this, I guess